src/interface/spy.jl

```diff
@@ -56,6 +56,8 @@ Arguments
 
 - **`canvas`** : The type of canvas that should be used for drawing.
 
+- **`zeros`** : Show zeros pattern instead of nonzeros (default).
```

Comment:
Just a confirmation; is this still the default? My understanding is that "by default it is `false`"?

I don't know if this matters if taking Plots.jl into account, but maybe `show_zeros` is easier to understand?

Reply:
The default of `spy(A)` is to show the sparsity pattern by coloring nonzeros entries (https://www.mathworks.com/help/matlab/ref/spy.html) in a sparse matrix (very useful in the context of finite differences, fem, applying boundary conditions, ...). So yes showing zeros is by default false. Agreed, another argument for renaming to `show_zeros` is that is does not shadow the builtin `zeros` from julia's stdlib. Showing zeros for a sparse matrix instead of nonzeros is not very used, but I find it occasionally useful !